src/hooks/useLogout.ts

```diff
@@ -20,8 +21,8 @@ const useLogout = (isGoogleLogin: boolean) => {
     return () => {
         if (isGoogleLogin) {
             signOut()
-            return
         }
+
         requestLogout()
```

Comment:
요거 순서를 requestLogout -> signOut 순으로 해주실수 있을까요?ㅠㅠㅠ

```suggestion
requestLogout()

if (isGoogleLogin) {
  signOut()
}
```